méli-mélo/2024-02-charts/js/charts.js

```diff
@@ -32,8 +32,10 @@ var componentName = "chart",
 				tableObserver,
 				elmChartType, supportedChartTypes, defaults,
 				chartType, chartOpts, chartData,
-				chartCntnr, canvasElm, detailsElm,
-				i18n, i18nText;
+				chartCntnr, canvasElm, tableContainer,
+				i18n, i18nText,
+				detailsOpen = "",
+				chartCntnrClass = "wb-chart-container";
```

Comment:
Will need more details how this use because I don't see it's function.

Reply:
As per our discussion, I have removed the class, but kept the `<div>`.